app/controllers/concerns/api/v2/lookup_keys_common_controller.rb

```diff
@@ -49,10 +49,10 @@ def smart_class_parameter_id?
 
   def find_smart_variable
```

Comment:
Is this whole method still needed? Feels like this leftover should have been removed, along with a few others in this file.

Reply:
yes, it makes sense to remove it, thanks @ekohl !